docs/threat-feeds.md

```diff
@@ -0,0 +1,58 @@
+| [Home](../README.md) |
+|----------------------|
+
+# Threat Feeds
+
+Using a wide variety of feed integrations that are available on the [Content Hub](https://fortisoar.contenthub.fortinet.com//list.html?contentType=all), you can seamlessly ingest feeds and get a normalized and aggregated view of the feeds on the Threat Feeds tab of **Threat Intel Management** <picture><source media="(prefers-color-scheme: dark)" srcset="./res/icon-chevron-light.svg"><source media="(prefers-color-scheme: light)" srcset="./res/icon-chevron-dark.svg"><img alt="Fallback image description" src="./res/icon-chevron-dark.svg"></picture> **Threat Intel Feeds**:
```

Comment:
image is not rendering <picture><source media="(prefers-color-scheme: dark)" srcset="./res/icon-chevron-light.svg"><

Reply:
<img width="1091" height="728" alt="image" src="https://github.com/user-attachments/assets/930f6f59-6b33-41e9-b84c-93d4696e6afa" />